packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -200,16 +205,14 @@ export const withPlatform = ({ parameters }) => {
           ></dds-masthead-composite>
         `
       : html`
-          <dds-masthead-container
-            platform="${ifNonNull(platformData.name)}"
+          <dds-masthead-composite
```

Comment:
Just wondering the reason behind the change here?

Reply:
this was to fulfill the "Remove all links from With platform story other than the Platform" requirement in the original issue. From what I could tell, the container will always render the links https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/masthead/masthead-container.ts#L86